deploy/jobs/jobs_contracts.go

```diff
@@ -125,6 +125,9 @@ func DeployJob(deploy *def.Deploy, do *def.DeployArgs, client *def.Client, resp
 		if binaryResponse.Error != "" {
 			return "", fmt.Errorf("Something went wrong when you were trying to link your binaries: %v", binaryResponse.Error)
 		}
+		if binaryResponse.Binary == "" {
+			return "", fmt.Errorf("Contract binary code missing")
```

Comment:
Can we add something like "Is your contract an abstract interface? You may not have unintentionally not implemented all abstracted inherited methods."

Reply:
Good point, done